### PR TITLE
PR for Issue 3056

### LINKED ISSUE #3056: Use bhInlineFilter on Role Management
If you ever have a long list of roles, locating a single one can be a nightmare.  We need to have a `bhInlineFilter` button on these pages.  You can see an example of one in the accounts management module code.

### PR DESCRIPTION
**Pull request by Scholldier**
This PR is to update two files to add inline filtering to the Roles Management module.

Closes #3056

### CODE REVIEW

**Comment by Scholldier** [comment]
So this code tests fine, EXCEPT that there is also a filter field above the 'Actions' -- which shouldn't be the case. Can you point me in the right direction to resolve that? I'll keep digging on my side but I don't see anything obvious yet.

**Comment by Scholldier** [comment]
I found it -- I added code to make it so that the actions column can't be filtered on and it tested correctly.

**Comment by Scholldier** [comment]
There are some things I need to fix according to the code climate test, which I'll review and fix tomorrow.

File: client/src/modules/roles/roles.ctrl.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ function RolesController($uibModal, Roles, Session, Modal, Notify, bhConstants)
     headerCellFilter : 'translate',
   }, {
     field : 'actions',
+    enableFiltering : false,
```
Comment (jniles) [inline client/src/modules/roles/roles.ctrl.js:95]: Glad you figured this out!  I suppose I should have mentioned it 😄

**Comment by jniles** [comment]
@Scholldier, no problem!  The code climate fixes should be straightforward.

Our tests on `master` are pretty flakey, but I've just landed a fix that should help... if it helps the other pull requests that are failing, I'll rebase this PR onto master while merging and we'll see if that solves the issue.

**Comment by Scholldier** [comment]
Haha, no worries -- it's good for me to put my troubleshooting hat on for learning sake. 👍

File: client/src/modules/roles/roles.ctrl.js (hunk 1 above)
Comment (Scholldier) [reply client/src/modules/roles/roles.ctrl.js:95]: Haha, no worries! It's good for me to learn through searching, trial and error, and such. 👍

**Comment by jniles** [comment]
Code looks good to me.  Downloaded and tested this and it behaves as excepted.  Good work.

bors r+

**Pull request merged**